dance/modules/single_modality/imputation/deepimpute.py

```diff
@@ -420,5 +422,15 @@ def score(self, true_expr, imputed_expr, mask=None, metric="MSE", test_idx=None)
         if metric == 'RMSE':
             return np.sqrt(F.mse_loss(true_target, imputed_target).item())
         elif metric == 'PCC':
-            corr_cells = np.corrcoef(true_target.cpu(), imputed_target.cpu())
-            return corr_cells
+            # corr_cells = np.corrcoef(true_target.cpu(), imputed_target.cpu())
+            # return corr_cells
+            return np.corrcoef(true_target.cpu()[~mask[test_idx]], imputed_target.cpu()[~mask[test_idx]])[0, 1]
+        elif metric == "MRE":
```

Comment:
mean relative error，Evaluate imputed data at different scales

Reply:
MRE paper: Hao M, Gong J, Zeng X, et al. Large scale foundation model on single-cell transcriptomics[J]. bioRxiv, 2023: 2023.05. 29.542705.

Reply:
In addition, if the prediction is correct, there should be no possibility that the actual value is 0, because the imputation task is performed